tests/scripts/thread-cert/v1_2_test_csl_transmission.py

```diff
@@ -71,6 +73,26 @@ def test(self):
         self.assertTrue(self.nodes[LEADER].ping(self.nodes[SSED_1].get_rloc()))
         self.simulator.go(5)
 
+        ssed_messages = self.simulator.get_messages_sent_by(SSED_1)
+        msg = ssed_messages.next_mle_message(mle.CommandType.CHILD_UPDATE_REQUEST)
+        msg.assertMleMessageContainsTlv(mle.CslChannel)
```

Comment:
should assert there is no csl channel tlv  as no csl channel is specified and it is the same as pan channel